calico/felix/fiptables.py

```diff
@@ -271,6 +271,8 @@ def _finish_msg_batch(self, batch, results):
                 _log.error("Non-retryable error from a combined batch, "
                            "splitting the batch to narrow down culprit.")
                 raise SplitBatchAndRetry()
+                # MD4: This introduces code to the actor and makes failure
```

Comment:
Yes, I think it is if we want to combine multiple iptables requests into one batch (which gives us a massive perf improvement) and, at the same time, isolate iptables failures to the particular request that caused them.

I think the latter is important in order to isolate failures to a particular rule chain or endpoint.  I think that makes felix a lot more robust in the face of mistakes in our iptables rule generation and the like.